src/content/docs/en/guides/deploy/google-cloud.mdx

```diff
@@ -35,67 +35,10 @@ Cloud Run is a serverless platform that allows you to run a container without ha
 
 3. Create a new service.
 
-#### Create the Dockerfile
 
-##### Dockerfile for static sites (example)
+#### Create Dockerfile & Build the Container
 
-This following Dockerfile will build your site and serve it using Apache HTTPD (default config) on port `80`.
-
-```docker title="Dockerfile"
-FROM node:lts AS build
-WORKDIR /app
-COPY . .
-RUN npm i
-RUN npm run build
-
-FROM httpd:2.4 AS runtime
-COPY --from=build /app/dist /usr/local/apache2/htdocs/
-EXPOSE 80
-```
-
-:::note[notes]
-If you're using a package manager other than `npm`, you'll need to change the commands:
-
-```docker title="Dockerfile" add={4} "pnpm"
-FROM node:lts AS build
-WORKDIR /app
-COPY . .
-RUN npm i -g pnpm@latest
-RUN pnpm i
-RUN pnpm run build
-# ...
-```
-:::
-
-##### Dockerfile for SSR sites (example)
-
-This following Dockerfile will build your site and serve it using Node.js (SSR adapter) on port `3000`.
-
-```docker title="Dockerfile"
-FROM node:lts AS runtime
-WORKDIR /app
-
-COPY . .
-
-RUN npm install
-RUN npm run build
-
-ENV HOST=0.0.0.0
-ENV PORT=3000
-EXPOSE 3000
-CMD node ./dist/server/entry.mjs
-```
-
-:::tip[Keep this in mind]
-These are just examples of Dockerfiles. You can customize them to your needs. For example, you could use another image, like `node:lts-alpine`.
-
-```docker title="Dockerfile" del={1} add={2}
-FROM node:lts AS runtime
-FROM node:lts-alpine AS runtime
-```
-:::
-
-#### Build the Container
+Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. You can find more [documentation on how to use Docker with Astro in our Docker Recipe](/en/recipes/docker/#creating-a-dockerfile).
```

Comment:
```suggestion
Before you can deploy your Astro site to Cloud Run, you need to create a Dockerfile that will be used to build the container. Find more information about [how to use Docker with Astro](/en/recipes/docker/#creating-a-dockerfile) in our recipe section.
```
I love a PR that deletes so many lines!  :partying_face: I'm just tightening up the call to action, then merging! Bye bye, all these lines!